app/lib/route53Client.ts

```diff
@@ -0,0 +1,11 @@
+import { Route53Client } from "@aws-sdk/client-route-53";
```

Comment:
I'm not sure that there's value in separating this out into its own file, since you only ever use it in `dns-server.ts`.  Let's move this into that file.

Reply:
Will do that